tardis/montecarlo/struct.py

```diff
@@ -61,6 +61,7 @@ class StorageModel(Structure):
         ('line_lists_tau_sobolevs_nd', c_int64),
         ('line_lists_j_blues', POINTER(c_double)),
         ('line_lists_j_blues_nd', c_int64),
+        ('line_lists_Edotlu', POINTER(c_double)),
```

Comment:
I think here's your bug.
The order matters for the definition of the storage_model struct fields. You defined them as
```
   double *line_lists_j_blues;
  double *line_lists_Edotlu;
   int64_t line_lists_j_blues_nd;
```